packages/governance/test/unitTests/test-paramGovernance.js

```diff
@@ -238,19 +238,21 @@ test('change multiple params', async t => {
     await E(publicFacet).getSubscription(),
   );
   const update1 = await notifier.getUpdateSince();
-  // constructing the fixture to deepEqual would complicate this with insufficient benefit
-  t.is(
-    // @ts-expect-error reaching into {} values
-    update1.value.current.Electorate.value.value[0].description,
-    'questionPoser',
-  );
-  t.like(update1, {
-    value: {
-      current: {
-        MalleableNumber: { type: 'nat', value: 602214090000000000000000n },
-      },
-    },
-  });
+  // This value isn't available synchronously
```

Comment:
Does this imply that the initial update has no useful values in it? That seems like a problem.  Can we publish it late, or read it later?

Reply:
Yes, yes and yes :)

This test is checking changing parameters works so the param might change before the "original" value is available. I'll try.